api/extension-guides/language-model.md

```diff
@@ -71,7 +71,7 @@ const craftedPrompt = [
 
 ## Send the language model request
 
-Once you've built the prompt for the language model, you need to select the language model by specifying the `vendor`, `id`, `family`, or `version` of the model you want to get. Currently, only `copilot-gpt-3.5-turbo` and `copilot-gpt-4` are supported. We expect that the list of supported models will grow over time. Once you have the model, you can send the request to it by using `sendRequest`.
+Once you've built the prompt for the language model, you need to select the language model by specifying the `vendor`, `id`, `family`, or `version` of the model you want to get. Currently, only `gpt-3.5-turbo` and `gpt-4` are supported. We expect that the list of supported models will grow over time. Once you have the model, you can send the request to it by using `sendRequest`.
```

Comment:
```suggestion
Once you've built the prompt for the language model, you need to select the language model by specifying the `vendor`, `id`, `family`, or `version` of the model you want to get. Learn more about these properties in the [API reference](/api/references/vscode-api#LanguageModelChat). Currently, only `gpt-3.5-turbo` and `gpt-4` are supported. We expect that the list of supported models will grow over time. Once you have the model, you can send the request to it by using `sendRequest`.
```

Reply:
Suggest adding a link to the API reference to get more details about the different properties.

Reply:
Thanks, also suggestion LGTM 😊